schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml

```diff
@@ -6,6 +6,27 @@
     <value value='2' nick='launch'/>
     <value value='3' nick='cycle-windows'/>
   </enum>
+  <enum id='org.gnome.shell.extensions.dash-to-dock.shiftClickAction'>
+    <value value='0' nick='skip'/>
+    <value value='1' nick='minimize'/>
+    <value value='2' nick='launch'/>
+    <value value='3' nick='cycle-windows'/>
+    <value value='4' nick='quit'/>
+  </enum>
+  <enum id='org.gnome.shell.extensions.dash-to-dock.middleClickAction'>
+    <value value='0' nick='skip'/>
+    <value value='1' nick='minimize'/>
+    <value value='2' nick='launch'/>
+    <value value='3' nick='cycle-windows'/>
+    <value value='4' nick='quit'/>
+  </enum>
+  <enum id='org.gnome.shell.extensions.dash-to-dock.shiftMiddleClickAction'>
+    <value value='0' nick='skip'/>
+    <value value='1' nick='minimize'/>
+    <value value='2' nick='launch'/>
+    <value value='3' nick='cycle-windows'/>
+    <value value='4' nick='quit'/>
+  </enum>
```

Comment:
Ah, you are right! I just tested it without the repetition.

Fixed in [0cd2bb009f54dd1acb84594fb4b3254a3f239bc0]